bin/sylius-cs-fixer

```diff
@@ -1,4 +0,0 @@
-#!/bin/bash
-
-php-cs-fixer fix . --config-file .php_cs
```

Comment:
what happened to php-cs-fixer? does this change with behat really involves php-cs-fixing? if not maybe another PR would be better place for discussion?